projects/plugins/jetpack/class.json-api-endpoints.php

```diff
@@ -1446,6 +1446,18 @@ function get_media_item_v1_1( $media_id, $media_item = null, $file = null ) {
 					$info = (object) $metadata['videopress'];
 				}
 
+				if ( isset( $info->rating ) ) {
+					$response['rating'] = $info->rating;
+				}
+
+				if ( isset( $info->display_embed ) ) {
+					$response['display_embed'] = (string) (int) $info->display_embed;
+					// If not, default to metadata (for WPCOM).
+				} elseif ( isset( $metadata['videopress']['display_embed'] ) ) {
+					// We convert it to int then to string so that (bool) false to become "0".
```

Comment:
😵‍💫 :)

Reply:
Yeah, looks a bit weird - we store the value in two different places (based on the environment).